test/config/multiplesettest.go

```diff
@@ -50,7 +50,7 @@ func (s *TestSuite) TestMultipleSet(t *testing.T) {
 
 		msValue := generateTimezoneName()
 
-		devicePath := gnmi.GetDevicePathWithValue(simulator.Name(), tzPath, msValue, proto.StringVal)
+		devicePath := gnmi.GetTargetPathWithValue(simulator.Name(), tzPath, msValue, proto.StringVal)
```

Comment:
Perhaps change the name of variables as well to target to avoid using two different terminology